content/en/docs/concepts/storage/volume-snapshots.md

```diff
@@ -22,61 +23,62 @@ This document describes the current state of `VolumeSnapshots` in Kubernetes. Fa
 
-`VolumeSnapshotContents` are resources in the cluster. `VolumeSnapshots` are requests for those resources. The interaction between `VolumeSnapshotContents` and `VolumeSnapshots` follow this lifecycle:
+`VolumeSnapshotContent` objects are resources in the cluster and `VolumeSnapshot` objects are requests for those resources. The interaction between `VolumeSnapshotContent` and `VolumeSnapshot` objects follows this lifecycle:
 
 ### Provisioning Volume Snapshot
 
 There are two ways snapshots may be provisioned: statically or dynamically.
 
 #### Static
-A cluster administrator creates a number of `VolumeSnapshotContents`. They carry the details of the real storage which is available for use by cluster users. They exist in the Kubernetes API and are available for consumption.
+A cluster administrator creates a number of `VolumeSnapshotContent` resources describing snapshots on the underlying storage that are available for use by cluster users. They exist in the Kubernetes API and are available for consumption.
 
 #### Dynamic
-When none of the static `VolumeSnapshotContents` the administrator created matches a user's `VolumeSnapshot`,
+When none of the static `VolumeSnapshotContent` resources the administrator created matches a user's `VolumeSnapshot`,
 the cluster may try to dynamically provision a volume snapshot specially for the `VolumeSnapshot` object.
-This provisioning is based on `VolumeSnapshotClasses`: the `VolumeSnapshot` must request a
+This provisioning is based on `VolumeSnapshotClass` resources: the `VolumeSnapshot` must request a
 [volume snapshot class](/docs/concepts/storage/volume-snapshot-classes/) and
 the administrator must have created and configured that class in order for dynamic
 provisioning to occur.
 
 ### Binding
 
-A user creates, or has already created in the case of dynamic provisioning, a `VolumeSnapshot` with a specific amount of storage requested and with certain access modes. A control loop watches for new VolumeSnapshots, finds a matching VolumeSnapshotContent (if possible), and binds them together. If a VolumeSnapshotContent was dynamically provisioned for a new VolumeSnapshot, the loop will always bind that VolumeSnapshotContent to the VolumeSnapshot. Once bound, `VolumeSnapshot` binds are exclusive, regardless of how they were bound. A VolumeSnapshot to VolumeSnapshotContent binding is a one-to-one mapping.
+A user creates, or has already created in the case of dynamic provisioning, a `VolumeSnapshot` with a specific set of parameters. A control loop watches for new `VolumeSnapshot` requests, finds a matching `VolumeSnapshotContent` (if possible), and binds them together. If a `VolumeSnapshotContent` was dynamically provisioned for the new `VolumeSnapshot`, the loop will always bind that `VolumeSnapshotContent` to the `VolumeSnapshot`. Once bound, `VolumeSnapshot` binds are exclusive, regardless of how they were bound. A `VolumeSnapshot` to `VolumeSnapshotContent` binding is a one-to-one mapping.
```

Comment:
Is this going into too much implementation detail talking about control loops?

Reply:
Similar description is there for PV/PVC binding too.

Reply:
I think we should keep the content as user-focused as possible and try not to leak implementation details into the docs.  Can we rephrase this to emphasize the behavior we want users to understand without talking about implementation?  The main points I see are:
* VolumeSnapshot and VolumeSnapshotContent binding is 1 to 1
* You can either request an pre-existing VolumeSnapshotContent, or dynamically provision one

Reply:
(And we should look into fixing up the wording for PVC page too)